addons/l10n_sa/models/account_move.py

```diff
@@ -0,0 +1,52 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+import base64
+
+from odoo import api, fields, models, _
+from odoo.exceptions import UserError
+
+
+class AccountMove(models.Model):
+    _inherit = 'account.move'
+
+    delivery_date = fields.Date(default=fields.Date.context_today, copy=False)
```

Comment:
Sorry for misleading, but I am afraid we will need l10n_sa_invoice in v15 and merge it into l10n_sa in master.